e2e/tests/functional/plugins/notebook/notebookWithCouchDB.e2e.spec.js

```diff
@@ -0,0 +1,232 @@
+        // 11) The shared worker event from 👆 POST request
+
+        requests = [];
+        await page.hover(`button:has-text("Add Tag")`);
+        await page.locator(`button:has-text("Add Tag")`).click();
+        await page.locator('[placeholder="Type to select tag"]').click();
+        await page.locator('[aria-label="Autocomplete Options"] >> text=Driving').click();
+        await page.waitForSelector('.c-tag__label:has-text("Driving")');
+        page.waitForLoadState('networkidle');
+        expect(filterNonFetchRequests(requests).length).toBeLessThanOrEqual(11);
+
+        requests = [];
+        await page.hover(`button:has-text("Add Tag")`);
+        await page.locator(`button:has-text("Add Tag")`).click();
+        await page.locator('[placeholder="Type to select tag"]').click();
+        await page.locator('[aria-label="Autocomplete Options"] >> text=Drilling').click();
+        await page.waitForSelector('.c-tag__label:has-text("Drilling")');
+        page.waitForLoadState('networkidle');
+        expect(filterNonFetchRequests(requests).length).toBeLessThanOrEqual(11);
+
+        requests = [];
+        await page.hover(`button:has-text("Add Tag")`);
+        await page.locator(`button:has-text("Add Tag")`).click();
+        await page.locator('[placeholder="Type to select tag"]').click();
+        await page.locator('[aria-label="Autocomplete Options"] >> text=Science').click();
+        await page.waitForSelector('.c-tag__label:has-text("Science")');
+        page.waitForLoadState('networkidle');
+        expect(filterNonFetchRequests(requests).length).toBeLessThanOrEqual(11);
+
+        // Delete all the tags
+        // Network requests are:
+        // 1) Send POST to mutate _delete property to true on annotation with tag
+        // 2) The shared worker event from 👆 POST request
+        // 3) Timestamp update on entry
+        // 4) The shared worker event from 👆 POST request
+        // This happens for 3 tags so 12 requests
+        requests = [];
+        await page.hover('.c-tag__label:has-text("Driving")');
+        await page.locator('.c-tag__label:has-text("Driving") ~ .c-completed-tag-deletion').click();
+        await page.waitForSelector('.c-tag__label:has-text("Driving")', {state: 'hidden'});
+        await page.hover('.c-tag__label:has-text("Drilling")');
+        await page.locator('.c-tag__label:has-text("Drilling") ~ .c-completed-tag-deletion').click();
+        await page.waitForSelector('.c-tag__label:has-text("Drilling")', {state: 'hidden'});
+        page.hover('.c-tag__label:has-text("Science")');
+        await page.locator('.c-tag__label:has-text("Science") ~ .c-completed-tag-deletion').click();
+        await page.waitForSelector('.c-tag__label:has-text("Science")', {state: 'hidden'});
+        page.waitForLoadState('networkidle');
+        expect(filterNonFetchRequests(requests).length).toBeLessThanOrEqual(12);
+
+        // Add two more pages
+        await page.click('text=Page Add >> button');
```

Comment:
a11y locator

Reply:
I was following your earlier `Page add` - is there another locator I should be using?